### DISCUSSION TITLE
libxmlsec1.so.1: symbol xmlIOFTPRead, version LIBXML2_2.4.30 not defined in file libxml2.so.2 with link time reference

### DISCUSSION THREAD

**Discussion by EPetrang** [latchset/mod_auth_mellon#62] [General]
Hello guys, 

I'm trying to get mellon working but I keep getting this error message, any idea on how to fix it?

` Cannot load /usr/lib64/httpd/modules/mod_auth_mellon.so into server: /lib64/libxmlsec1.so.1: symbol xmlIOFTPRead, version LIBXML2_2.4.30 not defined in file libxml2.so.2 with link time reference`

**Comment by simo5** [comment]
You have built mod_auth_mellon against a newer version of libxmlsec1 and trying to run it linking at runtime against an older version.
Update your libxmlsec1.so on the system or rebuild mod_auth_mellon (or more probably lasso) against the correct version.
